src/User32.Desktop/User32+INPUT.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="INPUT"/> nested type and its companion types <see cref="InputType"/> and <see cref="InputUnion"/>.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Used by <see cref="SendInput"/> to store information for synthesizing input events such as keystrokes, mouse movement, and mouse clicks.
+        /// </summary>
+        [StructLayout(LayoutKind.Sequential)]
+        public struct INPUT
+        {
+            public InputType type;
+
+            public InputUnion U;
```

Comment:
What an awful type. Thank you very much for figuring it out and contributing it!